### PR TITLE
 UI: Remove external dependencies and formatting

### PR DESCRIPTION
**Pull request by codenirvana**

### CODE REVIEW

File: dist/index.mustache [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (godfrzero) [inline dist/index.mustache]: Unnecessary empty line.

File: dist/index.mustache [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+<head>
```
Comment (godfrzero) [inline dist/index.mustache]: Indentation! The `head` tag is a child of `html`. We indent children.

File: dist/index.mustache [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    <title>Postman Collection Format {{ version }} Schema Documentation</title>
+    <meta charset="utf-8">
+    <link rel="shortcut icon" href="https://www.getpostman.com/favicon.ico">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    <link rel="stylesheet" href="../../css/docson.css">
+    <link rel="stylesheet" type="text/css" href="../../css/styles.css">
```
Comment (godfrzero) [inline dist/index.mustache]: Can these two CSS files be combined?

File: dist/index.mustache [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+                    <a href="http://schema.getpostman.com/json/collection/{{ version }}/collection.json" target=“_blank”>
+                        http://schema.getpostman.com/json/collection/{{ version }}/collection.json
+                    </a>
+                </p>
+            </div>
+        </div>
+    </div>
+
+    <div class="footer-container open-sans">
+        <span>&copy; 2018 Postdot Technologies, Inc.</span>
```
Comment (godfrzero) [inline dist/index.mustache]: We don't want to update the 2018 every year. Is there no way to get the current year in here?

File: dist/index.mustache [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                i[r] = i[r] || function () {
+                    (i[r].q = i[r].q || []).push(arguments)
+                }, i[r].l = 1 * new Date();
+                a = s.createElement(o),
+                    m = s.getElementsByTagName(o)[0];
+                a.async = 1;
+                a.src = g;
+                m.parentNode.insertBefore(a, m)
+            })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+            ga('create', 'UA-43979731-4', 'auto');
```
Comment (godfrzero) [inline dist/index.mustache]: TODO: Verify that this ID is still valid.

**Review by godfrzero — CHANGES_REQUESTED** [review]
I've refrained from suggesting anything that might trigger a refactor, and I'm assuming that only the header/footer have been touched.

To avoid repetitive work (and if it's sufficiently simple), I'd suggest using a layout.

File: dist/index.mustache (hunk 3 above)
Comment (codenirvana) [reply dist/index.mustache]: Probably not, The first one is `docson` library's CSS and its override and our custom styles are added to `styles.css`.

File: dist/index.mustache (hunk 5 above)
Comment (codenirvana) [reply dist/index.mustache]: Yes, verifed 👍

File: dist/table_of_contents.mustache [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+          </div>
+      </div>
+      <div class="footer-container open-sans">
+        <span>&copy;
+          <script type="text/JavaScript">document.write(new Date().getFullYear());</script>
+          Postdot Technologies, Inc.
+        </span>
+        <span>All Rights Reserved</span>
+      </div>
+      <script type="text/javascript">
```
Comment (kunagpal) [inline dist/table_of_contents.mustache]: This should be in the head: https://developers.google.com/tag-manager/quickstart

**Review by kunagpal — CHANGES_REQUESTED** [review]

**Review by kunagpal — APPROVED** [review]

**Pull request merged**